docs/DEVELOPMENT.md

```
@@ -42,4 +89,4 @@ python jobs/raw/public_api_raw_job.py --ENTITY_TYPE posts
 python jobs/bronze/public_api_bronze_job.py --ENTITY_TYPE posts
 python jobs/silver/public_api_silver_job.py --ENTITY_TYPE posts
 python jobs/gold/public_api_gold_job.py --ENTITY_TYPE posts
-```
+```
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)